scripts/update_and_rebuild_petsc_alt.sh

```diff
@@ -47,6 +47,6 @@ cd $SCRIPT_DIR
 sh $SCRIPT_DIR/update_and_rebuild_petsc.sh $* \
   --skip-submodule-update \
   --download-strumpack=0 \
-  --download-hypre=git://https://github.com/hypre-space/hypre.git
-
+  --download-hypre=git://https://github.com/hypre-space/hypre.git \
+  --with-64-bit-indices=0 \
```

Comment:
@loganharbour This is an example how to change 64-bit back to 32-bit. PETSc-alt will use 32-bit always.

Reply:
Oh, duh. I missed that - totally forgot it was a 1.  I'll add a 32 bit recipe now then.